### DISCUSSION TITLE
error: unrecognized subcommand 'scaffold'

### DISCUSSION THREAD

**Discussion by ontorojii** [loco-rs/loco#550] [Q&A]
I started Loco.rs today.  I could make new myapp, but encountered the problem when I did 'cargo loco generate scaffold post title:string content:text'.  I'm using Mac Mini Sonoma 14.4.1 cargo v1.77.2.
Pleasee let me know how to do then. 
---------------------------------------------
cargo loco generate scaffold post title:string content:text
    Finished dev [unoptimized + debuginfo] target(s) in 0.22s
     Running `target/debug/myapp-cli generate scaffold post 'title:string' 'content:text'`
error: unrecognized subcommand 'scaffold'

Usage: myapp-cli generate [OPTIONS] <COMMAND>

For more information, try '--help'.

**Comment by ShallmentMo** [comment]
I guess you are using `light-weight` starter, this project template doesn't  support `generate scaffold`. You can use '--help' to find all the options

**Reply by ontorojii** [reply]
Thank you.  I'll tray.

**Reply by joshka** [reply]
The fallback page instructions still have the scaffold command mentioned on them even when it's unavailable.

**Comment by saltedmocha** [comment]
Hi sorry for reviving this issue, this appears to be still a problem

I've tested both Sass app option with and without a database, scaffold option would be missing without a database. This doesn't seems to be pointed out on the starting docs unless I'm missing something here

This is the one without database:
<img width="1103" height="332" alt="image" src="https://github.com/user-attachments/assets/6dc3e9d5-407b-4a61-8235-8b2bd5c8f297" />

And this would be one with a database (I'm using sqlite for this example):
<img width="1103" height="383" alt="image" src="https://github.com/user-attachments/assets/efffccf7-6829-477e-9539-989846050a78" />

Both of these uses sass app with server side rendering and async background worker type
